frontend/packages/dev-console/src/components/pipelineruns/logs/__tests__/MultiStreamLogs.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { LogControls } from '@console/internal/components/utils';
+import { MultiStreamLogsProps, MultiStreamLogs } from '../MultiStreamLogs';
+import Logs from '../Logs';
+import { podData } from './logs-test-data';
+
+describe('MultiStreamLogs', () => {
+  let props: MultiStreamLogsProps;
+
+  beforeEach(() => {
+    props = {
+      taskName: 'step-oc',
+      resource: podData,
+    };
+  });
+
+  it('should exist', () => {
+    const wrapper = shallow(<MultiStreamLogs {...props} />);
+    expect(wrapper.isEmptyRender()).toBe(false);
+  });
+
+  it('should not render logs when containers is not present', () => {
+    props.resource.spec.containers = [];
+    const wrapper = shallow(<MultiStreamLogs {...props} />);
+    expect(wrapper.find(Logs)).toHaveLength(0);
+  });
+
+  it('should render inline loading based on logs completion', () => {
+    const wrapper = shallow(<MultiStreamLogs {...props} />);
+    expect(wrapper.find('.odc-multi-stream-logs__taskName--loading').exists()).toBe(false);
+    expect(wrapper.find('.odc-multi-stream-logs__taskName').text()).toBe('step-oc');
+  });
+
+  it('should render download and expand/collapse controls', () => {
+    const wrapper = shallow(<MultiStreamLogs {...props} />);
+    expect(wrapper.find(LogControls).exists()).toBe(true);
+  });
```

Comment:
This is just testing static content renders... not sure this is a good test.

Reply:
We can add shallow on LogControls and check for Expand/Collapse and Download button?

Reply:
That sounds like LogControls tests... not sure we should concern ourselves with it for MultiStreamLogs tests.

Reply:
Done